beacon-chain/p2p/connection_gater.go

```diff
@@ -84,19 +84,50 @@ func (s *Service) validateDial(addr multiaddr.Multiaddr) bool {
 // configureFilter looks at the provided allow lists and
 // deny lists to appropriately create a filter.
 func configureFilter(cfg *Config) (*multiaddr.Filters, error) {
+	//set list of private addresses
+	privateCIDRList := []string{
+		"10.0.0.0/8",
+		"172.16.0.0/12",
+		"192.168.0.0/16",
+		"100.64.0.0/10",
+		"169.254.0.0/16",
```

Comment:
How is this determined? 
IETF has set forth 3 ranges of private IPs
```
     10.0.0.0        -   10.255.255.255  (10/8 prefix)
     172.16.0.0      -   172.31.255.255  (172.16/12 prefix)
     192.168.0.0     -   192.168.255.255 (192.168/16 prefix)
```
Where did the other values come from?

https://tools.ietf.org/html/rfc1918

Reply:
Also, please add some commentary here. If you are using only the IETF values, you can mention it like
```go
// Private IP ranges as specified by rfc-1918
// See: https://tools.ietf.org/html/rfc1918
```

Reply:
Took them from this: [](https://en.wikipedia.org/wiki/Reserved_IP_addresses)
On second glance these are not exactly the same as private addresses. Was also looking at another issue referenced by this one, where the user was mentioning having to list the other two: #5663
Should I modify to just use the IETF values?

Reply:
OK I think we can try to figure out where those other ranges originated from. 

169.254.0.0/16 is specified here: https://tools.ietf.org/html/rfc3927
100.64.0.0/10 is specified here: https://tools.ietf.org/html/rfc6598

As long as we mention these and link to them in the commentary, then I think this would be acceptable set of IP ranges! 

Thanks for the PR, I learned a little bit about these private IP ranges :+1:

Reply:
Oh my bad! The link I wanted to post didn't show up (it was a wikipedia article that summarizes the links you posted: https://en.wikipedia.org/wiki/Reserved_IP_addresses). Thanks for digging these references up! Sorry about that. Anyways I will incorporate the commentary into the code.